homeassistant/components/device_tracker/traccar.py

```diff
@@ -41,6 +41,8 @@
     vol.Optional(CONF_PORT, default=8082): cv.port,
     vol.Optional(CONF_SSL, default=False): cv.boolean,
     vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
+    vol.Optional(CONF_MONITORED_CONDITIONS,
+                 default=[]): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
Is it impossible to validate what the conditions are?

Reply:
Yes, it is impossible because the monitored conditions (which in traccar server are called computed attributes) can be set to any name the user decides.